Gemfile

```diff
@@ -4,5 +4,5 @@ source "https://rubygems.org"
 gem "sinatra"
 gem 'sinatra-contrib'
 
-gem 'puma'
+gem 'puma', '>= 5.6.8'
```

Comment:
We don't generally specify versions in Gemfiles, please remove the constraint and run `bundle` again

Reply:
Thanks, done.